test/end2end_test.go

```diff
@@ -470,59 +471,73 @@ func listTestEnv() (envs []env) {
 // func, modified as needed, and then started with its startServer method.
 // It should be cleaned up with the tearDown method.
 type test struct {
-	t *testing.T
-	e env
-
+	// The following are setup in newTest().
+	t      *testing.T
+	e      env
 	ctx    context.Context // valid for life of test, before tearDown
 	cancel context.CancelFunc
 
-	// Configurable knobs, after newTest returns:
-	testServer              testpb.TestServiceServer // nil means none
-	healthServer            *health.Server           // nil means disabled
+	// The following knobs are for the server-side, and should be set after
+	// calling newTest() and before calling startServer().
+
+	// whether or not to expose the server's health via the default health service implementation.
+	enableHealthServer bool
+	// In almost all cases, one should set the 'enableHealthServer' flag on the
```

Comment:
'enableHealthServer' flag is not on the testServer now.

Reply:
Done.